docs/topics/pages.rst

```diff
@@ -415,9 +415,9 @@ You can make your model names more friendly to users of Wagtail by using Django'
         ...
 
         class Meta:
-            verbose_name = "homepage"
+            verbose_name = "Homepage"
```

Comment:
Django convention is for verbose_name to be lower-case - see #1578. (It may be that we're missing some code to re-capitalise this when outputting it on Wagtail templates, then...)